services/invitations/tests/unit/api/test_api_invitations.py

```diff
@@ -7,39 +7,43 @@
 import httpx
 from fastapi import status
 from fastapi.testclient import TestClient
+from hypothesis import HealthCheck, given, settings
+from hypothesis import strategies as st
 from models_library.api_schemas_invitations.invitations import (
     ApiInvitationContentAndLink,
 )
 from models_library.products import ProductName
 from simcore_service_invitations._meta import API_VTAG
 from simcore_service_invitations.api._invitations import INVALID_INVITATION_URL_MSG
-from simcore_service_invitations.invitations import (
+from simcore_service_invitations.services.invitations import (
     InvitationContent,
     InvitationInputs,
-    create_invitation_link,
+    create_invitation_link_and_content,
 )
 
 
+@settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+@given(invitation_input=st.builds(InvitationInputs))
```

Comment:
mind remembering these decorators?
given is the one that helps create the input
the @settings is for what already? this is something needed to use with pydantic?

Reply:
- settings tells hypothesis that the other fixtures (which are function-fixtures) can be re-used. In this case I can do that because I have a `client` and a constant with the `auth` header. I do not really need to clean them up every test